docs/installation/releases.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: Releases
+parent: Installation
+permalink: /installation/releases
+has_children: false
+nav_order: 1
+---
+# Releases
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+---
+<p>Releases of Meshery. See the Build and Release Strategy document for details.</p>
+
+<table class="responsive-table hover striped">
+  <tr>
+    <th class="centered">Version</th>
+    <th>Description</th>
+    <th>Release Date</th>
+  </tr>
+  <tr>
+    <td class="centered">0.1.0</td>
+    <td>Initial version of Meshery. Connects to Kubernetes and run preconfigured commands on Kubernetes with Istio</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.2.0</td>
+    <td>Ability to support running custom yaml on kubernetes with istio</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.3.0</td>
+    <td>Initial version with support of Meshery adapters and release of a pre-alpha version of Istio adapter</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.4.0</td>
+    <td>Meshery's User interface completely re-written using Reactjs and the backend updated as needed.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.5.0</td>
+    <td>Ability to view persisted results. Integration and support for Grafana charts and embedding panels in iframe</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.6.0</td>
+    <td>Pre-alpha Linkerd adapter. Ability to filter results.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.7.0</td>
+    <td>Pre-alpha Consul adapter. Support for deploying SMI Istio adapter and ensuring compatibility to apply SMI manifests through Meshery.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.8.0</td>
+    <td>Migrated away from embedded iframe grafana charts to using Chartjs and C3 for charting.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.9.0</td>
+    <td>Ability to import Grafana board json but integrating with Prometheus directly for metrics</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.10.0</td>
+    <td>Pre-alpha Octarine adapter. Support for static charts. Ability to record metrics for the static charts after the load test for later use.</td>
+    <td>June 24th, 2019</td>
+  </tr>
+  <tr>
+    <td class="centered">0.10.1</td>
+    <td>Patch for Alpine bug.</td>
+    <td>June 25th, 2019</td>
+  </tr>
+  <tr>
+    <td class="centered">0.11.0</td>
+    <td>Migrate from Configure Meshery to Settings page.</td>
+    <td>June 26th, 2019</td>
+  </tr>
+  <tr>
+    <td class="centered">0.12.0</td>
+    <td>Displace chartjs with billboardjs</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.13.0</td>
+    <td>Migrate from browser local storage to volume / filesystem storage</td>
```

Comment:
End all description elements with a period for consistency.

Reply:
Agree, yes.